src/main/java/edu/kis/powp/jobs2d/drivers/SelectAdditionalDriverMenuOptionListener.java

```diff
@@ -0,0 +1,32 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectAdditionalDriverMenuOptionListener implements ActionListener {
+    private DriverManager driverManager;
+    private Job2dDriver driver = null;
+    private boolean state =false;
+
+
+    public SelectAdditionalDriverMenuOptionListener(Job2dDriver driver, DriverManager driverManager) {
+        this.driverManager = driverManager;
+        this.driver = driver;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        System.out.println(e.paramString());
+        if (!state){
+            driverManager.addDriver(driver);
+        }else {
+            driverManager.removeDriver(driver);
+        }
+        state = !state;
+
+        DriverFeature.updateDriverInfo();
```

Comment:
now you should integrate your solution with: https://github.com/iis-powp-2020z/powp_jobs2d_projects/pull/2
addDriver and removeDriver implementations should trigger Observer notification automatically.

Reply:
Upstream merged in 1d214e7, integration pending

Reply:
Integrated with Observer in a4d8ca3